lib/SILOptimizer/Transforms/SimplifyCFG.cpp

```diff
@@ -663,7 +663,7 @@ bool SimplifyCFG::dominatorBasedSimplify(DominanceAnalysis *DA) {
 
   // Split all critical edges such that we can move code onto edges. This is
   // also required for SSA construction in dominatorBasedSimplifications' jump
-  // threading. It only splits new critical edges it creates by jump threading.
+  // threading. It only splits new critical edges and it creates by jump threading.
```

Comment:
This new ‘and’ changes meaning of the sentence.

Reply:
> Mostly looks good, thank you! I left some comments inline.

Hey @xedin I'm sorry about those errors I'm a little new to this, but I think have made all the changes you mentioned! Please let me know if there's anything else I need to do 😄